avatars/avatars.js

```diff
@@ -1492,28 +1513,21 @@ class Avatar {
     this.aimLeftFactorReverse = 1 - this.aimLeftFactor;
 
     const _updateHmdPosition = () => {
```

Comment:
If this is for local players, why not check `.isLocalPlayer`? I don't think it's a good idea to bleed update logic to the caller.